lib/runner/variant-reassignment.js

```diff
@@ -289,15 +384,31 @@ export default class VariantReassignment {
     if (products) {
       const skuToProductMap = this._createSkuToProductMap(products)
       const matchingProducts = productDraftSkus.map(sku => skuToProductMap.get(sku))
-      return _.uniq(matchingProducts)
+
+      // when there is a new non existing variant in product draft we will get undefined
```

Comment:
Split this one long sentence into smaller ones.

Reply:
Updated 8e56cf3